### PR TITLE
III-4456 Document onlineUrl and endpoint to update onlineUrl

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added model for `onlineUrl`
- Documented endpoint `PUT online-url` to update `onlineUrl`
- Documented endpoint `DELETE online-url` to delete `onlineUrl`

---

Ticket: https://jira.uitdatabank.be/III-4456

### CODE REVIEW

File: projects/uitdatabank/models/event-onlineUrl.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "$ref": "./common-string-uri.json",
+  "examples": [],
+  "title": "event.attendanceMode",
```
Comment (bertramakers) [inline projects/uitdatabank/models/event-onlineUrl.json:4]:
```suggestion
  "title": "event.onlineUrl",
```

File: projects/uitdatabank/reference/entry.json [hunk 2]
```diff
@@ -1853,6 +1853,63 @@
...
       }
     },
+    "/events/{eventId}/online-url": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "put": {
+        "summary": "online-url - update",
```
Comment (bertramakers) [inline projects/uitdatabank/reference/entry.json:1863]:
```suggestion
        "summary": "onlineUrl - update",
```

File: projects/uitdatabank/reference/entry.json [hunk 3]
```diff
@@ -1853,6 +1853,63 @@
...
+        "tags": [
+          "Events"
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content. The online url mode was successfully updated."
```
Comment (bertramakers) [inline projects/uitdatabank/reference/entry.json:1869]:
```suggestion
            "description": "No Content. The onlineUrl was successfully updated."
```

Comment (bertramakers) [reply projects/uitdatabank/reference/entry.json:1869]: (Note the "mode" that was left over from "attendance mode" I think)

**Comment by LucWollants** [comment]
@bertramakers I think this one can be merged, virtual events are ready.

**Review by bertramakers — APPROVED** [review]
I didn't review the DELETE yet, but looks good! 👍

**Comment by LucWollants** [comment]
@bertramakers everything got released to production on 11/05/2022

**Pull request merged**